src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PAY_PER_DAY = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+
+  if (days < SHORT_TERM) {
+    return days * PAY_PER_DAY;
```

Comment:
``days * PAY_PER_DAY;``
This logic is repeated and you can transfer it to a variable so as not to duplicate the code.